services/matrix/matrix.service.js

```diff
@@ -76,6 +85,15 @@ export default class Matrix extends BaseJsonService {
             name: 'server_fqdn',
             example: 'matrix.org',
           }),
+          queryParam({
+            name: 'fetchMode',
+            example: 'guest',
+            description: `If not specified, the default fetch mode is <code>guest</code> (except for matrix.org).`,
+            schema: {
+              type: 'string',
+              enum: ['guest', 'summary'],
```

Comment:
Could we define the options for this and then use it in both the `queryParamSchema` and the docs enum.

There's an example of what I mean here:

https://github.com/badges/shields/blob/18e17233c49cf94f9a32115c3fcdc439cb495086/services/gitlab/gitlab-release.service.js#L15-L79

Reply:
Done 4f6531defd595404d3aa83df5157e546bb9d43b9